proxy/envoy/testdata/lds-v0-egressrule-nomixer.json.golden

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
Same. Please compare to non egressrule to make sure egress rules do not affect LDS.